sdks/python/container/py310/base_image_requirements.txt

```diff
@@ -137,7 +137,7 @@ rpds-py==0.18.0
 rsa==4.9
 scikit-learn==1.4.2
 scipy==1.13.0
-shapely==2.0.3
+shapely==2.0.4
```

Comment:
this is a transitive dependency of cloudaiplatform and is low-risk upgrade for us